DOCUMENTATION.md

```diff
@@ -136,7 +136,9 @@ If you feel something is missing, not clear or could be improved, please don't h
         * [.login(credentials)](#resin.auth.login) ⇒ <code>Promise</code>
         * [.loginWithToken(authToken)](#resin.auth.loginWithToken) ⇒ <code>Promise</code>
         * [.isLoggedIn()](#resin.auth.isLoggedIn) ⇒ <code>Promise</code>
-        * [.createApiKey(name)](#resin.auth.createApiKey) ⇒ <code>Promise</code>
+        * [.createApiKey(name, [description])](#resin.auth.createApiKey) ⇒ <code>Promise</code>
+        * [.getAllApiKeys([options])](#resin.auth.getAllApiKeys) ⇒ <code>Promise</code>
+        * [.revokeApiKey(id)](#resin.auth.revokeApiKey) ⇒ <code>Promise</code>
```

Comment:
@pimterry give me a note if you want to namespace all these methods (but that would be breaking for `createApiKey`).

Also, in the UI we are going to update both the name and the description with the same action.
Would it make sense to provide `renameApiKey` & `noteApiKey` as  methods in here, regardless of how will we implement that in the UI?
PS: would love to hear a naming suggestion for an edit method that updates both fields and doesn't feel awkward.

Reply:
> give me a note if you want to namespace all these methods (but that would be breaking for createApiKey).

I think we should do this. Fortunately, MC is still beta, so we can! Let's go for it and break them out right here.

> Also, in the UI we are going to update both the name and the description with the same action.

`apiKey.update({ name: 'abc', description: 'def' })` would be fine I think. Where both are optional. For methods that'll be used more frequently externally it might be useful to have slightly more readable names for different calls, but I think for api keys we're the main way people will manage them, and that's fine for now.